src/reducers/sources.js

```diff
@@ -44,7 +44,10 @@ const State = makeRecord(
   }: SourcesState)
 );
 
-function update(state = State(), action: Action): Record<SourcesState> {
+export function update(
+  state: any = State(),
```

Comment:
I'm guessing that you could replace `any` with `Record<SourcesState>`. Can you try this?